### PR TITLE
Fix LiveApi.

### PR DESCRIPTION
**Pull request by totof3110**
Fixes https://github.com/scribejava/scribejava/issues/601.

Tested locally:
```
=== Windows Live's OAuth Workflow ===

Fetching the Authorization URL...
Got the Authorization URL!
Now go and authorize ScribeJava here:
https://login.live.com/oauth20_authorize.srf?client_id=MYCLIENTID&redirect_uri=MYREDIRECTURL&response_type=code&scope=wl.emails%20wl.signin
And paste the authorization code here
>>SOMECODE

Trading the Request Token for an Access Token...
Got the Access Token!
(if your curious it looks like this: Token[SOMETOKEN] )

Now we're going to access a protected resource...
Got it! Lets see what we found...

200
{
   "id": "SOMEID", 
   "name": "Christophe Levesque", 
   "first_name": "NAME", 
   "last_name": "NAME", 
   "gender": null, 
   "emails": {
      "preferred": "SOMEEMAIL", 
      "account": "SOMEEMAIL", 
      "personal": null, 
      "business": null
   }, 
   "locale": "en_US"
}

Thats it man! Go and build something awesome with ScribeJava! :)
```

### CODE REVIEW

File: scribejava-apis/src/main/java/com/github/scribejava/apis/LiveApi.java [hunk 1]
```diff
@@ -9,30 +9,27 @@
 import com.github.scribejava.core.utils.Preconditions;
 
 public class LiveApi extends DefaultApi20 {
+	private static final String AUTHORIZE_URL = "https://login.live.com/oauth20_authorize.srf?client_id=%s&redirect_uri=%s&response_type=code";
```
Comment (kullfar) [inline scribejava-apis/src/main/java/com/github/scribejava/apis/LiveApi.java]: do not use 'tabs' pls

File: scribejava-apis/src/test/java/com/github/scribejava/apis/examples/LiveExample.java [hunk 2]
```diff
@@ -1,8 +1,9 @@
 package com.github.scribejava.apis.examples;
 
 import java.util.Scanner;
-import com.github.scribejava.core.builder.ServiceBuilder;
+
 import com.github.scribejava.apis.LiveApi;
+import com.github.scribejava.core.builder.ServiceBuilder;
```
Comment (kullfar) [inline scribejava-apis/src/test/java/com/github/scribejava/apis/examples/LiveExample.java]: it makes unnecessary diff in the history

**Comment by kullfar** [comment]
I'll need test app and test user credentials to run the example my self. Could you provide them on https://github.com/scribejava/scribejava/blob/master/pom.xml#L48

**Comment by kullfar** [comment]
btw, I have merged a big changes to the master branch. Rebase pls. This Pull Request has conflicts for the moment

**Comment by totof3110** [comment]
@kullfar : PR rebased and updated.

**Comment by kullfar** [comment]
I still need test app and test user credentials to run the example myself. Could you provide them on https://github.com/scribejava/scribejava/blob/master/pom.xml#L48

**Comment by totof3110** [comment]
The test app credentials are already included in the source. To test it you can register for a free Live account at https://signup.live.com/signup if you don't already have one.

**Comment by kullfar** [comment]
Registering accounts in all provided by ScribeJava APIs is impossible for one not-payed person like me ;-)
btw. after review it's better to remove test app credentials from sources.

**Comment by kullfar** [comment]
Didn't received test user credentials. I'm still waiting for them to run the example by myself

**Comment by kullfar** [comment]
Didn't recieve test user credentials. just closing ;-(
